codebeat badge

korzonek opened the pull request:
Is it fine to add codebeat badge to README? 

codebeat is automated code review tool for Swift, Ruby & Go that helps get instant feedback on code quality. 

“Quick wins” suggested by codebeat could be a nice candidate for a pull request and help other developers become contributors.

FYI. To be fully open and honest. I’m co-founder of codebeat.

emilford commented:
Codebeat looks like a product that provides a very similar service to CodeClimate which we already use on our projects. If I missed something, feel free to re-open this PR for re-consideration. Thanks!

korzonek commented:
@emilford there are a few things that are not available in code climate and you can find in codebeat.

* _Multi-Language support for mobile and web applications_. We implemented all algorithms on our own. It means relatively quickly we can support more languages both for mobile and web. At this stage we analyse Swift, Ruby, Python and Go. It works! :)
* _Correct namespace reporting for Ruby_ Please compare reports for Sinatra. Code Climate is using flay and flog gem which are known to misreport namespaces if there is more than one in the same ruby file. You can see an example [here](https://codeclimate.com/github/sinatra/sinatra/Sinatra::Base) where some duplications should be reported for Sinatra::Templates instead of Sinatra::Base. In codebeat it looks us some time to do it right :) Please check (ratings for Sinatra)[https://codebeat.co/projects/github-com-sinatra-sinatra/ratings] in codebeat.
* _Quick Wins_. After analysing your project we will suggest you issues in code with the biggest overall impact on your project's health. Try to refactor these hot spots first and see how your GPA improves.
* _Compare Changes_. In the stream of Recent Changes you will be able see what exactly caused your classes to improve or degrade.
* _Performance_ We are super proud of our backend. Analysers are really fast :)

Our mission at codebeat is to develop a set of tools that will help manage code quality within organisations. By code quality, I mean code robustness, security, efficiency and reusability. We started with code robustness :) 

We implemented all algorithms on our own so we could search for duplicates across all repositories within the organisation. We can apply static analysis to find security vulnerabilities.

Please give it a try. I hope you will like it. It would be fantastic if you could share your feedback and ideas how we could improve codebeat.

BTW… we are about to release support for pull requests.
